data_folder/work_preferences.yaml

```diff
@@ -42,6 +44,18 @@ company_blacklist:
 title_blacklist:
   - word1
   - word2
+  - C++
+  - C#
+  - .NET
+
 
 location_blacklist:
-  - Brazil
\ No newline at end of file
+  - Brazil
+
+job_applicants_threshold:
```

Comment:
there are moved to config.py. revert this from line 55

Reply:
you a right, missed it